colin-api/src/colin_api/models/filing.py

```diff
@@ -1297,8 +1297,17 @@ def add_filing(cls, con, filing: Filing) -> int:
     @classmethod
     def _get_last_ar_filed_date(cls, header: dict, business: dict):
         filing_year = header.get('filingYear')
-        recognition_dt = datetime.datetime.fromisoformat(business.get('business').get('foundingDate')).date()
-        last_ar_filed_dt = f'{filing_year}-{recognition_dt.month}-{recognition_dt.day}'
+        founding_date_str = business.get('business').get('foundingDate')
+
+        # If the founding date has `-00:00`, replace it with `+00:00` before passing to the function
+        if '-00:00' in founding_date_str:
+            founding_date_str = founding_date_str.replace('-00:00', '+00:00')
```

Comment:
I am not really sure about this. Approving it since this function is used only for Business AR sync